pkg/controller/migcluster/validation.go

```diff
@@ -87,38 +87,40 @@ func (r ReconcileMigCluster) validate(cluster *migapi.MigCluster) error {
 func (r ReconcileMigCluster) validateURL(cluster *migapi.MigCluster) error {
 	// Not needed.
 	if cluster.Spec.IsHostCluster {
+		log.V(4).Info("skipping url validation as this is a host cluster",
```

Comment:
"skipping url validation for host cluster"

Reply:
Updated the PR